symengine/functions.h

```diff
@@ -965,6 +965,66 @@ class Erfc : public OneArgFunction
 //! Canonicalize Erfc:
 RCP<const Basic> erfc(const RCP<const Basic> &arg);
 
+class Erfinv : public OneArgFunction
+{
+    /*   Inverse Error Function. The erfinv function is defined as:
+
+    .. math ::
+        \mathrm{erf}(x) = y \quad \Rightarrow \quad \mathrm{erfinv}(y) = x
+     **/
+public:
+    IMPLEMENT_TYPEID(ERFINV)
+    Erfinv(const RCP<const Basic> &arg) : OneArgFunction(arg)
+    {
+        SYMENGINE_ASSIGN_TYPEID()
+        SYMENGINE_ASSERT(is_canonical(arg))
+    }
+    bool is_canonical(const RCP<const Basic> &arg) const;
+    virtual RCP<const Basic> create(const RCP<const Basic> &arg) const;
+};
+
+RCP<const Basic> erfinv(const RCP<const Basic> &arg);
+
+class Erfcinv : public OneArgFunction
+{
+    /*   Inverse Complementary Error Function. The erfcinv function is defined
+    as:
+
+    .. math ::
+        \mathrm{erfc}(x) = y \quad \Rightarrow \quad \mathrm{erfcinv}(y) = x
+     **/
+public:
+    IMPLEMENT_TYPEID(ERFCINV)
+    Erfcinv(const RCP<const Basic> &arg) : OneArgFunction(arg)
+    {
+        SYMENGINE_ASSIGN_TYPEID()
+        SYMENGINE_ASSERT(is_canonical(arg))
+    }
+    bool is_canonical(const RCP<const Basic> &arg) const;
+    virtual RCP<const Basic> create(const RCP<const Basic> &arg) const;
+};
+
+RCP<const Basic> erfcinv(const RCP<const Basic> &arg);
+
+class Erf2 : public TwoArgFunction
```

Comment:
I don't think we need a special function for this. instead of `erf2(a, b)` we can just return `erf(a)-erf(b)`